packages/ui/src/molecules/Gift/Gift.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+import type { ReactNode } from 'react'
+
+import Icon from '../../atoms/Icon'
+export interface GiftProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
Just an extra line break to separate the concerns:
```suggestion
import Icon from '../../atoms/Icon'

export interface GiftProps extends HTMLAttributes<HTMLDivElement> {
```